docs/getting_started.md

```diff
@@ -0,0 +1,325 @@
+# Getting started
+
+## Data generation process
+
+The data generation process uses this analogy: generated data flows from source to sink.
+
+To generate data is then necessary to define:
+
+* `source`: what data is generated, eg. data model
+* `sink`: where data is sent to, eg. ES index
+* `flow`: how data is transmitted, eg. how fast or how much?
+* `schema`: fields definition, eg. ECS 8.2.0
+
+Each of the above is handled by its own REST API endpoint. An arbitray
+number of sources, sinks, flows and schemas can be defined on the same
+server.
+
+## Install
+
+Currently Geneve is packaged only for [Homebrew](https://brew.sh), you
+need first to install the Geneve tap
```

Comment:
It's really awesome you've got it on Homebrew 🎉 , but would be nice to have local setup/install instructions as well for folks running/testing locally that may want to make changes on the fly.

Reply:
That's a "Developers guide"! I'll add it.